packages/backend/src/config/config.local.ts

```diff
@@ -52,6 +52,7 @@ export function getLocalConfig(): Config {
     tvl: tvlEnabled && {
       alchemyApiKey: getEnv('ALCHEMY_API_KEY'),
       etherscanApiKey: getEnv('ETHERSCAN_API_KEY'),
+      arbiscanApiKey: getEnv('ARBISCAN_API_KEY'),
```

Comment:
one more important thing, after we merge it people will need to update their .env files for local development, please inform especially Bartek and Luca about this

Reply:
you know what? let's make this optional for now, because we will hide all this functionality behind the feature flag anyway